aws/resource_aws_emr_cluster.go

```diff
@@ -1424,8 +1424,7 @@ func flattenEBSConfig(ebsBlockDevices []*emr.EbsBlockDevice) *schema.Set {
 		if ebs.VolumeSpecification.VolumeType != nil {
 			ebsAttrs["type"] = *ebs.VolumeSpecification.VolumeType
 		}
-		ebsAttrs["volumes_per_instance"] = 1
-
+		ebsAttrs["volumes_per_instance"] = len(ebsBlockDevices)
```

Comment:
Is it possible to specify multiple `ebs_config` (multiple configurations with 1 or more volumes per instance)? e.g.

```terraform
  master_instance_group {
    instance_type = "m4.large"

    ebs_config {
      size                 = 32
      type                 = "gp2"
      volumes_per_instance = 2
    }

    ebs_config {
      size                 = 50
      type                 = "gp2"
      volumes_per_instance = 2
    }
  }
```

If so, the `volumes_per_instance` needs to be calculated per configurations type.

Reply:
I don't think this is an issue here. The `volumes_per_instance` is an attribute of `ebs_config`.

Reply:
Ok, I get what you're saying.
I updated the PR and test case
but we can not dealing with 2 same ebs_config config like this
```
  master_instance_group {
    instance_type = "m4.large"

    ebs_config {
      size                 = 32
      type                 = "gp2"
      volumes_per_instance = 2
    }

    ebs_config {
      size                 = 32
      type                 = "gp2"
      volumes_per_instance = 1
    }
  }
```

Reply:
I think not being able to handle that last configuration there is okay -- seems reasonable to say that should have one `ebs_config` with `volumes_per_instance = 3` 👍  Also thank you for adding the additional configuration to the test case!